src/account_management.rs

```diff
@@ -0,0 +1,163 @@
+use crate::contacts::initial_sync;
+use crate::{
+    ACCOUNTS_DIR, Config, ensure_accounts_dir, get_account_store_path, list_accounts, open_store,
+};
+use anyhow::{Result, anyhow};
+use presage::Manager;
+use std::path::Path;
+
+pub async fn create_account_cli(account_name: String, device_name: String) -> Result<()> {
+    use futures::{channel::oneshot, future};
+    use presage::libsignal_service::configuration::SignalServers;
+    use qr2term;
+
+    println!("Creating account '{}'...", account_name);
+
+    let accounts = list_accounts()?;
+    if accounts.contains(&account_name) {
+        return Err(anyhow::anyhow!("Account '{}' already exists", account_name));
+    }
+    ensure_accounts_dir()?;
+
+    let account_dir = format!("{}/{}", ACCOUNTS_DIR, account_name);
+    let store_path = get_account_store_path(&account_name);
+
+    tokio::fs::create_dir_all(&account_dir).await?;
+
+    let _ = std::fs::remove_file(&store_path);
+    let store = open_store(&store_path).await?;
+
+    let (tx, rx) = oneshot::channel();
+    let name = account_name.clone();
+    let (manager_result, _err) = future::join(
+        Manager::link_secondary_device(store, SignalServers::Production, device_name, tx),
+        async move {
+            match rx.await {
+                Ok(url) => {
+                    println!("Scan the QR code to link the device to account '{}'!", name);
+                    qr2term::print_qr(url.as_ref()).expect("QR generation failed");
+                    println!("You can also use the URL: {url}");
+                }
+                Err(err) => println!("Error while linking device: {err}"),
+            }
+        },
+    )
+    .await;
+
+    let mut manager = match manager_result {
+        Ok(manager) => {
+            println!("Device linked successfully! Syncing contacts...");
+            manager
+        }
+        Err(e) => return Err(anyhow!("Error while linking device: {e}")),
+    };
+
+    initial_sync(&mut manager).await?;
+    let mut config = Config::load();
+    config.set_current_account(account_name.clone());
+    config
+        .save()
+        .map_err(|e| anyhow::anyhow!("Failed to save config: {}", e))?;
```

Comment:
```suggestion
        .map_err(|e| anyhow!("Failed to save config: {e}"))?;
```
Nit pick. Already imported. Plus variables can be put inside `{}` and it should be done this way whenever possible. (Function calls or anything that requires dots like struct fields or methods cannot be inlined like this. Run clippy with `clippy::uninlined_format_args` to check this.